doc/Language/rb-nutshell.pod6

```diff
@@ -140,7 +140,7 @@ $person.name   # Perl 6
 To call a method whose name is not known until runtime:
 
 =for code :lang<ruby>
-object.send(methodname, args);  # Ruby
+object.public_send(methodname, args);  # Ruby
 =for code
 my $object; my Str $methodname; my @args; ...;
 $object."$methodname"(@args);   # Perl 6
```

Comment:
Ruby's `#send` works with private and public methods. New to Perl 6, but I'm pretty sure this only works with public methods.